flutter/shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -206,6 +208,33 @@ void PlatformViewChannel::OnDispose(
   result->Success();
 }
 
+void PlatformViewChannel::OnOffset(
+    const EncodableValue* arguments,
+    std::unique_ptr<MethodResult<EncodableValue>>&& result) {
+  auto* map = std::get_if<EncodableMap>(arguments);
+  if (!map) {
+    result->Error("Invalid arguments");
+    return;
+  }
+
+  EncodableValueHolder<int> view_id(map, "id");
+  EncodableValueHolder<double> left(map, "left");
+  EncodableValueHolder<double> top(map, "top");
+
+  if (!view_id || !left || !top) {
+    result->Error("Invalid arguments");
+    return;
+  }
+  PlatformView* view = FindViewById(*view_id);
+  if (!view) {
+    result->Error("Can't find view id");
+    return;
+  }
+  view->Offset(*left, *top);
+
+  result->Success(*arguments);
```

Comment:
Why use `arguments` for success response?  is the dart side expecting to get it back?

Reply:
```java
        private void offset(@NonNull MethodCall call, @NonNull MethodChannel.Result result) {
          Map<String, Object> offsetArgs = call.arguments();
          try {
            handler.offset(
                (int) offsetArgs.get("id"),
                (double) offsetArgs.get("top"),
                (double) offsetArgs.get("left"));
            result.success(null);
          } catch (IllegalStateException exception) {
            result.error("error", detailedExceptionString(exception), null);
          }
        }
```
Android implementation is passing null.
probably, you don't need passing arguments.

Reply:
Changed to pass null. thank you!

Reply:
Note: I think that when writing a `MethodResult` response, you should look at the corresponding Dart implementation.
In this case, maybe you should take a look [here](https://github.com/flutter-tizen/flutter-tizen/blob/e3fa9d40b4273e0bb3b78c50a5fbdfdf25b5f927/packages/flutter_tizen/lib/src/services/platform_views.dart#L240).

Thanks you for response.